projectFilesBackup/.idea_modules/dr-elephant.iml

```diff
@@ -0,0 +1,304 @@
+<module type="JAVA_MODULE" version="4">
```

Comment:
Remove this file. These are intellij files. You can add them to .gitignore file to avoid people including it in the future.